go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/apache/arrow/go/v13 v13.0.0-20230630125530-5a06b2ec2a8e
 	github.com/armon/go-metrics v0.4.1
 	github.com/bits-and-blooms/bloom/v3 v3.5.0
-	github.com/bytedance/sonic v1.9.1
+	github.com/bytedance/sonic v1.12.2
```

Comment:
If it has nothing to do with the issue, don't merge it together...

Reply:
this is to fix this issue after disable cgo: https://github.com/bytedance/sonic/issues/697